test/registries.conf

```diff
@@ -1,10 +1,9 @@
 # Note that changing the order here may break tests.
-[registries.search]
-registries = ['docker.io', 'quay.io', 'registry.fedoraproject.org']
+unqualified-search-registries = ['docker.io', 'quay.io', 'registry.fedoraproject.org']
 
-[registries.insecure]
-registries = []
-
-#blocked (docker only)
-[registries.block]
-registries = []
+# In Nov. 2020, Docker rate-limits image pulling.  To avoid hitting these
+# limits while testing, always use the google mirror for qualified and
+# unqualified `docker.io` images.
+# Ref: https://cloud.google.com/container-registry/docs/pulling-cached-images
```

Comment:
Isn't there a `[[registry]]` missing?

Reply:
Easily possible, I may have misinterpreted your https://github.com/containers/podman/pull/7965#discussion_r509202113 comment.  Though...it seems to work without it, is that indication of a bug?